package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-shared",
-  "version": "7.2.0",
+  "version": "7.2.2",
```

Comment:
7.2 0 if no current release, or 7.2.1. Check current releases 😄

Reply:
changed to 7.2.0